salt/modules/file.py

```diff
@@ -5836,6 +5954,9 @@ def check_file_meta(
             if serange and serange != current_serange:
                 changes["selinux"] = {"range": serange}
 
+    if any([ignore_ordering, ignore_whitespace, ignore_comment_characters]):
+        return has_changes, changes
```

Comment:
We're returning a different data type then what is normally expected. Just concerned this might cause issue.